bigframes/session/bq_caching_executor.py

```diff
@@ -112,7 +112,7 @@ def execute(
         max_results: Optional[int] = None,
     ) -> executor.ExecuteResult:
         if use_explicit_destination is None:
-            use_explicit_destination = bigframes.options.bigquery.allow_large_results
+            use_explicit_destination = bigframes.options.compute.allow_large_results
```

Comment:
Need to use the bigquery option if the compute option is not set.

```suggestion
            allow_large_results = bigframes.options.compute.allow_large_results
            if allow_large_results is None:
                allow_large_results = bigframes.options.bigquery.allow_large_results
            use_explicit_destination = allow_large_results
```

Since this is going to be used in many places, please make a helper function somewhere in `bigframes._config` for this.

Reply:
added one in options, not sure if that's the best place though.